### PR TITLE
Adding the structure of urlConfiguration in rewriteRuleActionSet

### PR DESCRIPTION
**Pull request by abjai**
### Latest improvements:

These changes are already been approved in issue https://github.com/Azure/azure-rest-api-specs/pull/7030 but could not be merged because of NRP release. These changes are targeted to merge in September milestone.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
@KrisBash from ARM already reviewed these changes here: https://github.com/Azure/azure-rest-api-specs/pull/7030

File: specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json [hunk 1]
```diff
@@ -1843,6 +1850,23 @@
       },
       "description": "Header configuration of the Actions set in Application Gateway."
     },
+    "ApplicationGatewayUrlConfiguration": {
+      "properties": {
+        "modifiedPath": {
+          "type": "string",
+          "description": "It is the url path which user has provided for url rewrite. Null means no path will be updated. Default value is null."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json]: "It is" is kind of redundand, most descriptions in spec start would start with "The URL path..."

File: specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json [hunk 2]
```diff
@@ -1843,6 +1850,23 @@
...
+        },
+        "modifiedQueryString": {
+          "type": "string",
+          "description": "It is the query string of url which user has provided for url rewrite. Null means no query string will be updated. Default value is null."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json]: "It is" is kind of redundand, most descriptions in spec start would start with "The query string..."

**Review by anton-evseev — APPROVED** [review]
Looks good except a few nitpicks about descriptions

File: specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json (hunk 2 above)
Comment (abjai) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json]: Resolved as per new commit

**Comment by number213** [comment]
@sergey-shandar could you please review and merge?

**Pull request merged**